index.html

```diff
@@ -158,8 +158,11 @@ <h2>Contact me </h2>
             </div>
             <form action="https://formspree.io/f/mzboldye" method="post">
                 <ul>
+                    <li><label for="name">Enter your name</label></li>
                     <li><input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Enter Your name"  required></li>
+                    <li><label for="email">Enter your email</label></li>
                     <li><input type="email" id="email" name="user_email" placeholder="Enter your email" required></li>
+                    <li><label for="msg">Write your message here</label></li>
                     <li><textarea name="message" id="msg" cols="30" rows="10" maxlength = "500" placeholder="Write your message here" required></textarea></li>
                     <li><button type="submit">Get It Touch</button></li>
                 </ul>
```

Comment:
- [ ] Nice job adding the label tag :+1:  After checking I think is not implemented well. Please kindly fix them to have better web accessibility. :smile: 

- [ ] Please, kindly fix the issue with the color contrast for your navigation menu below to improve the web accessibility of your page to improve the user experience. :+1: 

![Screenshot from 2022-07-13 13-12-03](https://user-images.githubusercontent.com/6615394/178741849-4de907ce-9ade-420c-9468-76a5bff20fcc.png)

- [ ] Please I would like to check the button's contrast. You can check it with [Aditus](https://www.aditus.io/button-contrast-checker/)
![Screenshot from 2022-07-13 13-14-49](https://user-images.githubusercontent.com/6615394/178742814-3849b60f-1668-4882-b732-b8c1426ad3c6.png)

- [ ]  Please check your single project technologies list Contrast Errors because it is very low .

![Screenshot from 2022-07-13 13-27-27](https://user-images.githubusercontent.com/6615394/178744978-a3427f60-4baf-4938-8d85-6eea6a3fad94.png)

- [ ] I would like to recommend you put a URL link for all the `href ` attributes because that will improve your web accessibility :+1: 

![Screenshot from 2022-07-13 13-28-29](https://user-images.githubusercontent.com/6615394/178745853-5b50a108-7847-48b6-a430-7d32b21c29d9.png)


- [ ] [OPTIONAL] I would like to recommend you include [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) in your inputs since you don't have label tags.

- [ ] [OPTIONAL] Please install an icon in your browser's tab. so that you will be able to check your website accessibility status.  You can follow [this link](https://stackoverflow.com/questions/4888377/how-to-add-a-browser-tab-icon-favicon-for-a-website).

![Screenshot from 2022-07-13 13-34-42](https://user-images.githubusercontent.com/6615394/178746537-afedcdb6-0194-49f6-8d41-745f56a53969.png)